src/lib/constants/postCard-config.ts

```diff
@@ -0,0 +1,28 @@
+import {
```

Comment:
check the filename - should be `kebab-case`,
also, it's not about config, this is `mock-data.ts` :)

Reply:
fixed